reward_extraction/behavior_cloning/behavior_cloning.py

```diff
@@ -0,0 +1,191 @@
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.functional as F 
+from torch.utils.data import Dataset, DataLoader
+from typing import Tuple
+from robot.data import RoboDemoDset
+from robot.base_robot import XArmTaskSpaceEnv
+# export PYTHONPATH="${PYTHONPATH}:/home/xarm/Documents/JacobAndDavin/test/rewardlearning-robot"
+device = 'cuda' if torch.cuda.is_available() else 'cpu'
+class BCNet(nn.Module):
+    def __init__(self) -> None:
+        super(BCNet, self).__init__()
+        self.fc1 = nn.Linear(3, 30)
+        self.fc2 = nn.Linear(30, 300)
+        self.fc4 = nn.Linear(300, 30)
+        self.fc3 = nn.Linear(30, 3)
+        self.relu = nn.LeakyReLU()
+    
+    def forward(self, x):
+        x = self.fc1(x)
+        x = self.relu(x)    
+        x = self.fc2(x)
+        x = self.relu(x)
+        x = self.fc4(x)
+        x = self.relu(x)
+        x = self.fc3(x)
```

Comment:
you probably wanna tanh the output so the range is squished to become 0 and 1.

Reply:
pinned you on Slack for this issue